app/views/moderations/actions_panel.html.erb

```diff
@@ -141,7 +141,7 @@
         value="<%= rating_level[0] %>"
         name="rating_vote_<%= rating_level[0] %>"
         id="js__rating__vote__<%= rating_level[0] %>"
-        class="level-rating-button <%= " selected" if @rating_vote&.rating == rating_level[0].to_f %>"
+        class="level-rating-button <%= " selected" if @rating_vote&.rating&.to_i == rating_level[0] %>"
```

Comment:
I'm least certain of this change -  I'm not opposed to ignoring the rubocop rule being called by the linter here.

Reply:
I'm also wondering whether `rating.round(0)` were the better option (I can never remember if to_i is truncation or not).

Reply:
Can we add a custom Rubocop rule once we decide which one is the way to go?

Reply:
I don't want to be the guy that mentions "types", but if think the rule would be "when converting a float to an integer, you will normally want to call round, not to_i", while ignoring the totally expected behavior of parsing strings as integers. I'd be  pleasantly surprised if inferring the type of a variable during static analysis were easy. 

Good

     "123".to_i => 123
     (123.6).round(0) => 124
     123.to_i => 123
     123.round(0) => 123 (weird looking, but correct)

Suspicious (not sure that it's wrong if it's the desired behavior, it usually isn't).

     (123.6).to_i => 123